pkg/gateways/http/transfer/list_transfer.go

```diff
@@ -10,14 +10,29 @@ import (
 )
 
 func (s *Handler) ListTransfers(w http.ResponseWriter, r *http.Request) {
-	token := r.Header.Get("Authorization")
+	w.Header().Set("content-type", "application/json")
+	header := r.Header.Get("Authorization")
+
+	token, err := CheckCredential(header)
 
 	l := s.logger.WithFields(log.Fields{
 		"module": "https",
-		"method": "processLogin",
+		"method": "processTransfer",
+		"token":  token,
 	})
-	w.Header().Set("content-type", "application/json")
 	e := errorStruct{l: l, token: token, w: w}
+
+	if err != nil {
+		l.WithFields(log.Fields{
+			"type":  http.StatusBadRequest,
+			"time":  domain2.CreatedAt(),
+			"token": token,
+			"where": "checkcredential",
```

Comment:
well, if you need to inform on which function or method the error occurred may the error message is not so clear, don't you agree?